pkg/kallsyms/bpf/privatedata.bpf.c

```diff
@@ -9,13 +9,21 @@ const volatile u64 socket_ino = 0;
 
 // initialized by this ebpf program
 volatile u64 tracer_pid_tgid = 0;
+
 struct {
 	__uint(type, BPF_MAP_TYPE_ARRAY);
 	__uint(max_entries, 1);
 	__type(key, int); // index: zero
 	__type(value, u64); // private_data
 } ig_private_data SEC(".maps");
 
+struct {
+	__uint(type, BPF_MAP_TYPE_ARRAY);
+	__uint(max_entries, 1);
+	__type(key, int); // index: zero
+	__type(value, u64); // f_op
+} ig_f_op SEC(".maps");
+
```

Comment:
What about using the same map with a structure containing private data and f_op? That'd make the handling easier as there is a single map, also readStructFileFields could return this structure.